### PR TITLE
Express Post JSON | Yang An Boada

### PR DESCRIPTION
**Pull request by yanganboada**
post json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const grades = [];
+
+const getJson = app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+const jsonParse = express.json(getJson);
+app.use(jsonParse, (req, res) => {
+  grades.push(req.body);
+  res.sendStatus(201);
```
Comment (thebearingedge) [inline express-post-json/index.js:13]: Each new grade object should receive a new id property. The server should respond with the created object too.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 1 above)
Comment (yanganboada) [reply express-post-json/index.js]: Express Post JSON Req Res Updates | Yang An Boada
9c1c63e

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const grades = [];
+
+const getJson = app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+const jsonParse = express.json(getJson);
+app.use(jsonParse, (req, res) => {
+  const idNum = grades.length + 1;
```
Comment (thebearingedge) [inline express-post-json/index.js:12]: There should be another variable near grades named `nextId`. That will be your id

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 2 above)
Comment (yanganboada) [reply express-post-json/index.js]: Express Post JSON Id Update | Yang An Boada
e3214e5

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**